bin/utils/file-utils.sh

```diff
@@ -10,7 +10,27 @@
 # Copyright IBM Corporation 2020
 ################################################################################
 
-#TODO LATER - provide flag that toggles all functions to error if they exit non-zero?
+# TODO LATER - anyway to do this better?
```

Comment:
Not sure a better way, one thing we may be able to try to consolidate the variable names `INSTALL_DIR`, `ZOWE_ROOT_DIR`, and `ROOT_DIR`. But seems not the scope of this PR.

Reply:
Agreed - created https://github.com/zowe/zowe-install-packaging/issues/1373 to cover